core/parameters/src/config_store.rs

```diff
@@ -57,6 +57,8 @@ static CONFIG_DIFFS: &[(ProtocolVersion, &str)] = &[
     (77, include_config!("77.yaml")),
     (78, include_config!("78.yaml")),
     (79, include_config!("79.yaml")),
+    // Equalizes the cost of reading trie nodes for first and subsequent accesses
```

Comment:
Don't need comments here. First because these comments don't actually describe the protocol version in full (if somebody adds more changes to 81.yaml;) and second because it is actually pretty easy to look at the file and understand the changes from the source of truth.